packages/blocks/src/store/selectors.js

```diff
@@ -554,7 +554,7 @@ export function getGroupingBlockName( state ) {
 export const getChildBlockNames = createSelector(
 	( state, blockName ) => {
 		return map(
-			filter( state.blockTypes, ( blockType ) => {
+			getBlockTypes( state ).filter( ( blockType ) => {
```

Comment:
We're using `getBlockTypes( state )` because `state.blockTypes` is an object by default:

https://github.com/WordPress/gutenberg/blob/3284f343a4c69a74a1e29eac03a14d0d2c6c9cc1/packages/blocks/src/store/reducer.js#L93

so we'd like to call `Object.values()` on it, but that's exactly what `getBlockTypes()` does:

https://github.com/WordPress/gutenberg/blob/93c5efd7ebbebf0ae73247866e5f12589887ede0/packages/blocks/src/store/selectors.js#L71